test/drenv/__main__.py

```
@@ -245,8 +315,23 @@ def add_stress_test_command(sp):
     )
 
 
-def add_command(sp, name, func, help=None, envfile=True):
-    parser = sp.add_parser(name, help=help)
+def add_command(
+    sp,
+    name,
+    func,
+    help=None,
```

Ruff (0.16.0)

[error] Function argument `help` is shadowing a Python builtin

(A002)